src/selectors/entities/users.ts

```diff
@@ -474,6 +473,38 @@ export function makeGetProfilesForReactions(): (state: GlobalState, reactions: A
     );
 }
 
+export const getProfilesInChannel: (state: GlobalState, channelId: $ID<Channel>, skipInactive: boolean) => Array<UserProfile> = createSelector(
```

Comment:
turns out I didnt need these two new selectors and I can get by with just using `makeGetProfiles...`